### PR TITLE
EPMLABSBRN-698 add rest api for audiometry

### PR DESCRIPTION
**Pull request by ElenaSpb**
[EPMLABSBRN-698](https://jira.epam.com/jira/browse/EPMLABSBRN-698)

add rest api for audiometry

### CODE REVIEW

File: api-contract/api.raml [hunk 1]
```diff
@@ -6,6 +6,29 @@ baseTestUri: http://10.66.216.143:8081/api
 mediaType:
   - application/json
 
+# AUDIOMETRY
+/audiometrics
+    get:
+        queryParameters:
+            locale:
```
Comment (lifeart) [inline api-contract/api.raml:13]: why we need locale here? locale

File: api-contract/api.raml [hunk 2]
```diff
@@ -6,6 +6,29 @@ baseTestUri: http://10.66.216.143:8081/api
...
+                required: false
+                type: String
+                example: "ru-ru"
+                default: "ru-ru"
+        responses:
+            200:
+                body:
+                    example: !include samples/audiometrics.json
+
+/audiometryHistory
```
Comment (lifeart) [inline api-contract/api.raml:23]: let's don't use camelCase in endpoint names,
`audiometryHistory` -> `audiometry-history`, 

anyway I'm not celarly understand meaning of this API

File: api-contract/api.raml (hunk 1 above)
Comment (ElenaSpb) [reply api-contract/api.raml:13]: yes, as we have audiometryType=SPEECH

Comment (ElenaSpb) [reply api-contract/api.raml:13]: in this case content would be provide by locale

File: api-contract/api.raml (hunk 2 above)
Comment (ElenaSpb) [reply api-contract/api.raml:23]: to save speech diagnostic results

Comment (ElenaSpb) [reply api-contract/api.raml]: done: audiometryHistory -> audiometry-history

**Pull request merged**